### PR TITLE
Update installation instructions on Fedora

### PR DESCRIPTION
**Pull request by callahad**
This replaces #9137 

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
* [x] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)
* [x] Code style is correct (run the [linters](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#code-style))

### CODE REVIEW

**Comment by callahad** [comment]
However this gets squash-merged, we should make sure that Git preserves the metadata and attribution to the original contributor for #9137

**Comment by callahad** [comment]
Cancelled CI. This is a doc only change. Newsfile passes, which is all we really care about here.

File: INSTALL.md [hunk 1]
```diff
@@ -151,29 +151,15 @@ sudo pacman -S base-devel python python-pip \
...
 ```sh
 sudo dnf install libtiff-devel libjpeg-devel libzip-devel freetype-devel \
-                 libwebp-devel tk-devel redhat-rpm-config \
-                 python3-virtualenv libffi-devel openssl-devel
+                 libwebp-devel libxml2-devel libxslt-devel libpq-devel \
+                 python3-virtualenv libffi-devel openssl-devel python3-devel
 sudo dnf groupinstall "Development Tools"
 ```
 
-Installing prerequisites on CentOS 7 or Fedora<=25:
```
Comment (clokep) [inline INSTALL.md:163]: Are these just old versions that aren't supported anymore?

**Review by clokep — APPROVED** [review]
I think this looks fine! I did not test it, but @callahad confirmed he did.

File: INSTALL.md (hunk 1 above)
Comment (callahad) [reply INSTALL.md:163]: Yep, they've been out of support for a while (CentOS 7 since Dec 31, Fedora 25 since 2017)

**Pull request merged**